### DISCUSSION TITLE
怎么配置某个域名不走缓存，每次都重新解析

### DISCUSSION THREAD

**Discussion by 804959783** [kkkgo/PaoPaoDNS#153] [问答和帮助]
群晖docker安装，有动态公网ip，做了动态域名解析，在本地环境通过域名访问不到本地的服务，但是用手机流量却可以通过域名访正常访问，ping了下域名发现解析的ip还是之前的，实际公网ip已经变了，怎么能设置这个域名不走缓存，或者有什么其他的解决办法，求帮忙

**Comment by kkkgo** [comment]
1、本地域名建议重写为局域网IP，不用绕一圈公网IP再访问
2、你托管的DDNS域名可能位于境外导致连接性不太好
3、实时更新请参考`RULES_TTL`功能

**Reply by 804959783** [reply]
感谢老哥，我先试下RULES_TTL，主要是我不知道域名指向局域网ip怎么加端口，我用域名访问都要加端口的

**Reply by 804959783** [reply]
麻烦再问下老哥，我这测试脚本报错，而且权威dns的ip也不是我的ip，实际ip已经变了，这是什么问题
![image](https://github.com/kkkgo/PaoPaoDNS/assets/35617062/b9e0a21d-4d1f-4480-883c-218889afa521)

**Reply by kkkgo** [reply]
>感谢老哥，我先试下RULES_TTL，主要是我不知道域名指向局域网ip怎么加端口，我用域名访问都要加端口的


你把你的服务器内网端口和外网端口保持一致，不就可以了。


dnshijack表明你的网络环境存在DNS劫持，具体跑debug.sh分析。

**Reply by kkkgo** [reply]
另外版本更新了，可以拉一下最新版本镜像
